lib/git_reflow/git_helpers.rb

```diff
@@ -10,7 +10,7 @@ def git_root_dir
     end
 
     def git_editor_command
-      @git_editor_command ||= GitReflow::Config.get('core.editor')
```

Comment:
Removed the memoize because GitReflow::Config already seems to cache the only "slow" code, and I got wrongly memoized values in the two `it` blocks in the spec